public/javascripts/SVValidate/src/Main.js

```diff
@@ -262,13 +262,13 @@ function Main (param) {
         lng: param.language,
         debug: false
     }, function(err, t) {
-        if(param.init !== "noInit") {
+        if (param.init !== "noInit") {
             defineValidateConstants();
             _initUI();
 
             if (param.hasNextMission) {
                 _init();
-            } else {
+            } else if (!param.hasNextMission || svv.missionLength === 0) {
```

Comment:
The change on this line doesn't actually do anything! We'd only get here if `param.hasNextMission` was false, meaning that `!param.hasNextMission` is guaranteed to be true, meaning that `!param.hasNextMission || svv.missionLength === 0` is also true... Which is the same thing as just saying `} else {`!